### PR TITLE
Add debian and ubuntu supported versions

### PR DESCRIPTION
**Pull request by vcerenu**
## Description

This Pr adds supported versions of Debian and Ubuntu into Puppet install process.
Related issue https://github.com/wazuh/wazuh-puppet/issues/958
### Docs building
- [x] Compiles without warnings.
### Code formatting and web optimization
- [ ] Uses three spaces indentation.
- [ ] Adds or updates meta descriptions accordingly.
- [ ] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [ ] Uses present tense, active voice, and semi-formal registry.
- [ ] Uses short, simple sentences.
- [ ] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

**Review by teddytpc1 — APPROVED** [review]

File: source/deployment-options/deploying-with-puppet/setup-puppet/install-puppet-master.rst [hunk 1]
```diff
@@ -40,16 +40,16 @@ Create a symbolic link between the installed binary file and your default binary
...
-+------------+---------+--------+---------+------+--------+---------+--------+-------------+
-| **Debian** | jessie  | wheezy | stretch | sid                                            |
-+------------+---------+--------+---------+----------------------------------+-------------+
++----------------------------------------------------------------------------+--------------------------+
+| The releases supported by the manifest to install Wazuh are as follows:                               |
++------------+---------+--------+---------+--------+----------+----------+--------+-------------+-------+
+| **Ubuntu** | precise | trusty | vivid   | wily   | xenial   | yakketi  | bionic | focal fossa | jammy |
++------------+---------+--------+---------+--------+----------+----------+--------+-------------+-------+
+| **Debian** | jessie  | wheezy | stretch | buster | bullseye | bookworm |                              |
++------------+---------+--------+---------+--------+----------+----------+------------------------------+
```
Comment (javimed) [inline source/deployment-options/deploying-with-puppet/setup-puppet/install-puppet-master.rst:49]:
```suggestion
The manifest supports the following releases to install wazuh.

**Ubuntu**

-  Jammy Jellyfish
-  Focal Fossa
-  Bionic Beaver
-  Yakkety Yak
-  Xenial Xerus
-  Wily Werewolf
-  Vivid Vervet
-  Trusty Tahr
-  Precise Pangolin

**Debian**

-  bookworm
-  bullseye
-  buster
-  stretch
-  jessie
-  wheezy
```

File: source/deployment-options/deploying-with-puppet/setup-puppet/install-puppet-master.rst [hunk 2]
```diff
@@ -40,16 +40,16 @@ Create a symbolic link between the installed binary file and your default binary
...
 
 .. note::
```
Comment (javimed) [inline source/deployment-options/deploying-with-puppet/setup-puppet/install-puppet-master.rst:51]:
```suggestion
```

File: source/deployment-options/deploying-with-puppet/setup-puppet/install-puppet-master.rst [hunk 3]
```diff
@@ -40,16 +40,16 @@ Create a symbolic link between the installed binary file and your default binary
...
-  ``Ubuntu Focal Fossa`` is supported since version 4.0.4
+  ``Ubuntu focal fossa`` is supported since version 4.0.4
```
Comment (javimed) [inline source/deployment-options/deploying-with-puppet/setup-puppet/install-puppet-master.rst:54]:
```suggestion
```

**Review by javimed — CHANGES_REQUESTED** [review]

File: source/deployment-options/deploying-with-puppet/setup-puppet/install-puppet-master.rst (hunk 1 above)
Comment (vcerenu) [reply source/deployment-options/deploying-with-puppet/setup-puppet/install-puppet-master.rst]: done
image

File: source/deployment-options/deploying-with-puppet/setup-puppet/install-puppet-master.rst (hunk 2 above)
Comment (vcerenu) [reply source/deployment-options/deploying-with-puppet/setup-puppet/install-puppet-master.rst]: Done
image

**Review by rauldpm — CHANGES_REQUESTED** [review]
This does not seem consistent with 4.8.0 as wazuh-puppet does not specify Debian bookworm as a supported OS 

https://github.com/wazuh/wazuh-puppet/blob/4.8.0/metadata.json

**Comment by teddytpc1** [comment]
@javimed, I agreed with @davidjiglesias to add this change to 4.8.0 RC4. We can merge this PR.

**Review by javimed — APPROVED** [review]

**Pull request merged**